[Bug]: `dev` branch: bad sampler name: `Use same sampler`, TypeError: 'NoneType' object is not subscriptable

### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What happened?

bad sampler name: Use same sampler，TypeError: 'NoneType' object is not subscriptable
When I update the SDxl branch, it often reports errors

### Steps to reproduce the problem

When I update the SDxl branch, it often reports errors


### What should have happened?

nor

### Version or Commit where the problem happens

https://github.com/AUTOMATIC1111/stable-diffusion-webui/commit/a2360de3f343ab4deec180e0266aca69fae3a78c

### What Python version are you running on ?

Python 3.10.x

### What platforms do you use to access the UI ?

Windows

### What device are you running WebUI on?

Nvidia GPUs (RTX 20 above)

### Cross attention optimization

xformers

### What browsers do you use to access the UI ?

Microsoft Edge

### Command Line Arguments

```Shell
--medvram --theme dark
```


### List of extensions

sd-webui-prompt-all-in-one
sd-webui-controlnet
sd-webui-additional-networks
multidiffusion-upscaler-for-automatic1111

### Console logs

```Shell
1.Traceback (most recent call last):
  File "G:\stable-diffusion-webui\venv\lib\site-packages\gradio\routes.py", line 442, in run_predict
    output = await app.get_blocks().process_api(
  File "G:\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1392, in process_api
    result = await self.call_function(
  File "G:\stable-diffusion-webui\venv\lib\site-packages\gradio\blocks.py", line 1097, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "G:\stable-diffusion-webui\venv\lib\site-packages\anyio\to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "G:\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 877, in run_sync_in_worker_thread
    return await future
  File "G:\stable-diffusion-webui\venv\lib\site-packages\anyio\_backends\_asyncio.py", line 807, in run
    result = context.run(func, *args)
  File "G:\stable-diffusion-webui\venv\lib\site-packages\gradio\utils.py", line 703, in wrapper
    response = f(*args, **kwargs)
  File "G:\stable-diffusion-webui\modules\ui_extensions.py", line 157, in extension_table
    ext.read_info_from_repo()
  File "G:\stable-diffusion-webui\modules\extensions.py", line 61, in read_info_from_repo
    self.from_dict(d)
  File "G:\stable-diffusion-webui\modules\extensions.py", line 45, in from_dict
    setattr(self, field, d[field])
TypeError: 'NoneType' object is not subscriptable

2.   Traceback (most recent call last):
      File "G:\stable-diffusion-webui\modules\call_queue.py", line 58, in f
        res = list(func(*args, **kwargs))
      File "G:\stable-diffusion-webui\modules\call_queue.py", line 37, in f
        res = func(*args, **kwargs)
      File "G:\stable-diffusion-webui\modules\txt2img.py", line 63, in txt2img
        processed = processing.process_images(p)
      File "G:\stable-diffusion-webui\modules\processing.py", line 685, in process_images
        res = process_images_inner(p)
      File "G:\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\batch_hijack.py", line 42, in processing_process_images_hijack
        return getattr(processing, '__controlnet_original_process_images_inner')(p, *args, **kwargs)
      File "G:\stable-diffusion-webui\modules\processing.py", line 802, in process_images_inner
        samples_ddim = p.sample(conditioning=p.c, unconditional_conditioning=p.uc, seeds=p.seeds, subseeds=p.subseeds, subseed_strength=p.subseed_strength, prompts=p.prompts)
      File "G:\stable-diffusion-webui\modules\processing.py", line 1089, in sample
        return self.sample_hr_pass(samples, decoded_samples, seeds, subseeds, subseed_strength, prompts)
      File "G:\stable-diffusion-webui\modules\processing.py", line 1115, in sample_hr_pass
        self.sampler = sd_samplers.create_sampler(img2img_sampler_name, self.sd_model)
      File "G:\stable-diffusion-webui\modules\sd_samplers.py", line 30, in create_sampler
        assert config is not None, f'bad sampler name: {name}'
    AssertionError: bad sampler name: Use same sampler
```


### Additional information

_No response_

--- a/modules/txt2img.py
+++ b/modules/txt2img.py
@@ -42,7 +42,7 @@ def txt2img(id_task: str, prompt: str, negative_prompt: str, prompt_styles, step
         hr_resize_x=hr_resize_x,
         hr_resize_y=hr_resize_y,
         hr_checkpoint_name=None if hr_checkpoint_name == 'Use same checkpoint' else hr_checkpoint_name,
-        hr_sampler_name=hr_sampler_name,
+        hr_sampler_name=None if hr_sampler_name == 'Use same sampler' else hr_sampler_name,
         hr_prompt=hr_prompt,
         hr_negative_prompt=hr_negative_prompt,
         override_settings=override_settings,

